### ISSUE TITLE
ROCm 2.7 rocm-smi no more display AvgPwr

### ISSUE THREAD

**Issue by ye-luo** [RadeonOpenCompute/ROCm#867]
ROCm 2.7 shows AvgPwr N/A
```
 ========================ROCm System Management Interface========================
================================================================================
GPU  Temp   AvgPwr  SCLK    MCLK    Fan    Perf  PwrCap  VRAM%  GPU%  
0    45.0c  N/A     808Mhz  350Mhz  8.63%  auto  250.0W    0%   0%    
================================================================================
==============================End of ROCm SMI Log ==============================
```
I can get a number in ROCm 2.6.

PS: the extra space on the first line is ugly.

**Comment by kentrussell** [comment]
The space issue has been addressed.
The power issue is a firmware issue. I'm working on it but don't know how long it will take to get into a release. If you want to do some hacking to get it working, you can get the 2.3-2.6 release (any of them work) and replace the vega20/vega20_smc.bin firmware file with the one from the older release and that will take care of it.

**Comment by kentrussell** [comment]
I have a fix for the kernel to address it for newer SMU FW . I am hoping to get it into 2.8

**Comment by kentrussell** [comment]
Fix available in 2.8. Turns out we had a couple of issues that got exposed, but they're addressed in 2.8 now.
